tests/raftstore/transport_simulate.rs

```diff
@@ -208,8 +211,143 @@ impl FilterFactory for Isolate {
                         }];
         }
         vec![box PartitionFilter {
-                 node_ids: vec![node_id],
+                 node_ids: vec![self.node_id],
```

Comment:
the bug is in this line.

an isolated node should neither send message out, nor receive any message, so `Isolate` generate a `DropPacketFilter` for the node itself, preventing send out any message, and also generate `PartitionFilter` for other nodes, preventing send message to the isolated node.

it's a mistake that I use a wrong node_id here, thanks @BusyJay for pointing out.

if I just test Filter as a black box, nothing is wrong.
it's `Isolate` wrong which is a FilterFactory, FilterFactory is something like a combinator, so the test case is a bit complex, but if I don't fix this bug, the test case for isolate would fail, of course.

@siddontang